libs/clients/passports/src/lib/passportsApi.service.ts

```diff
@@ -0,0 +1,269 @@
+import { Auth, AuthMiddleware, User } from '@island.is/auth-nest-tools'
+import { XRoadConfig, ConfigType } from '@island.is/nest/config'
+import { Injectable, Inject } from '@nestjs/common'
+import {
+  IdentityDocumentApi,
+  IdentityDocumentResponse,
+  PreregistrationApi,
+} from '../../gen/fetch'
+import {
+  Gender,
+  IdentityDocument,
+  IdentityDocumentChild,
+  Passport,
+  PreregisterResponse,
+  PreregistrationInput,
+} from './passportsApi.types'
+import PDFDocument from 'pdfkit'
+import getStream from 'get-stream'
+import type { Logger } from '@island.is/logging'
+import { LOGGER_PROVIDER } from '@island.is/logging'
+import { ApolloError } from 'apollo-server-express'
+import isBefore from 'date-fns/isBefore'
+import differenceInMonths from 'date-fns/differenceInMonths'
+import { ExpiryStatus } from './passportsApi.types'
+
+const LOG_CATEGORY = 'passport-service'
+
+@Injectable()
+export class PassportsService {
+  constructor(
+    @Inject(LOGGER_PROVIDER)
+    private logger: Logger,
+    @Inject(XRoadConfig.KEY)
+    private xroadConfig: ConfigType<typeof XRoadConfig>,
+    private identityDocumentApi: IdentityDocumentApi,
+    private preregistrationApi: PreregistrationApi,
+  ) {}
+
+  handleError(error: any, detail?: string): ApolloError | null {
```

Comment:
```suggestion
  handleError(error: unknown, detail?: string): null {
```
This function isn't handling anything... Why not just let the functions below crash and burn by themselves?